oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/utils/index.js

```diff
@@ -1,13 +1,17 @@
+import { useEffect, useRef } from "react";
+import axios from "axios";
+
 import _sortBy from "lodash/sortBy";
 import _concat from "lodash/concat";
 import _has from "lodash/has";
 import _partition from "lodash/partition";
 import _isEmpty from "lodash/isEmpty";
+import _isFunction from "lodash/isFunction";
 
 export function sortByStatusCode(requests) {
   // TODO: why we are checking status_code of first request
   // instead of just checking if requests array is empty 
-  if (!_has(requests[0], "status_code")) {
+  if (_isEmpty(requests) || !_has(requests[0], "status_code")) {
```

Comment:
If there are atleast some requests, they always have some status_code

Reply:
Removed and fixed